File: Codechef folder/Level 1 to 2/imdb.cpp
//IMDB
#include<iostream>
using namespace std;
int main()
{
    int t,n,x,i,maxr=0;
    cin>>t;
    while(t--)
    {
        cin>>n>>x;
        int arr[n][2];
        for(i=0;i<n;i++)
        {
            cin>>arr[i][0]>>arr[i][1];
            if(arr[i][0]<=x)
            {
                if(maxr<arr[i][1])
                    maxr=arr[i][1];
            }
        }
        cout<<maxr<<endl;
        maxr=0;
    }
}

File: Codechef folder/Level 1 to 2/imdb.py
#IMDB
t=int(input())
maxr=0
lt=[]
while t:
    n,x=map(int,input().split())
    i=0
    for i in range (0,n):
        lt.append(input().split())
    i=0
    while i<n:
            if int(lt[i][0])<=x:
                if maxr<int(lt[i][1]):
                    maxr=int(lt[i][1])
            i=i+1
    print(maxr,end="\n")
    maxr=0
    lt=[]
    t=t-1

